transactions/reset-epoch/2023/jun-22/README.md

```diff
@@ -0,0 +1,221 @@
+# Reset Epoch
+
+> June 22nd, 2023
+
+Any time the network is sporked, the protocol state is manually initialized via the bootstrapping process.
+Therefore, we need to re-synchronize the FlowEpoch smart contract each time the network is sporked, using the [ResetEpoch](../../../../templates/reset_epoch_with_end_staking_auction.cdc) transaction.
+
+The ResetEpoch transaction however needs to be proposed by the staking contract account and payed by the service account.
+Currently, the [multi-sign vercel app](https://flow-multisig-git-service-account-onflow.vercel.app/mainnet) does not support transactions with different proposer and payer hence we have to execute this via the old method using the flow-cli.
+
+## Sequence of signing: 
+1. Vishal generates the Tx
+2. Round one - Signing for staking account
+   1. Josh signs with flow-staking.json
+   2. Dmitrii Signs with flow-staking.json
+   3. Emerson Signs with flow-staking.json
+   4. Vishal Signs with flow-staking.json
+3. Roune two - - Signing for service account
+   1. Josh signs with flow.json
+   2. Dmitrii Signs with flow.json
+   3. Emerson Signs with flow.json
+   4. Vishal Signs with flow.json
+4. Vishal submits the transaction
+
+## Prerequisites
+1. Latest version of the [flow-cli](https://developers.flow.com/tools/flow-cli).
+2. [Gcloud cli](https://cloud.google.com/sdk/docs/install)
+3. Contributor privilege to this repo.
+4. Checkout this repo locally.
+5. Open a terminal and go into the directory `service-account`
+
+## Dapper Labs Builds
+
+```sh
+flow transactions build ./templates/reset_epoch_with_end_staking_auction.cdc \
+  --config-path flow-staking.json \
+  --network mainnet \
+  --args-json "$(cat "./transactions/reset-epoch/2023/jun-22/arguments.json")" \
+  --proposer 0x8624b52f9ddcd04a \
+  --proposer-key-index 5 \
+  --authorizer 0x8624b52f9ddcd04a \
+  --payer 0xe467b9dd11fa00df \
+  --gas-limit 1000000 \
```

Comment:
I think we should probably increase the gas limit to 10000000 just to be safe since now we're calculating rewards in addition to moving tokens between buckets.